shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -258,7 +258,7 @@ - (void)dispatchPointerDataPacket:(std::unique_ptr<flutter::PointerDataPacket>)p
 }
 
 - (flutter::PlatformViewIOS*)iosPlatformView {
-  FML_DCHECK(_shell);
+  FML_DCHECK(_shell) << "Cannot run iosPlatformView without a shell";
```

Comment:
@gaaclarke @ColdPaleLight I've added a more verbose message here
![](https://user-images.githubusercontent.com/20296911/151455352-643aafe1-6f70-40f1-845a-6cf2ec3bac3c.png)

Looking at the trace above, the FML_DCHECK(_shell) is failing, so I can't use engine.shell anyhow since this would fail the FML_DCHECK too.
https://github.com/flutter/engine/blob/70798c3174c9bcfc6d63deb9e6e9a105ebd0949f/shell/platform/darwin/ios/framework/Source/FlutterEngine.mm#L232

Maybe the message here would be better if it was similar to https://github.com/flutter/engine/blob/70798c3174c9bcfc6d63deb9e6e9a105ebd0949f/shell/platform/darwin/ios/framework/Source/FlutterEngine.mm#L1091 i.e 
```Cannot run iosPlatformView without a shell (engine possibly not run).```